src/github.com/matrix-org/dendrite/clientapi/routing/password.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2017 Vector Creations Ltd
+// Copyright 2017 New Vector Ltd
+//
```

Comment:
This file (and its contents) got created in 2018. And now we just have New Vector